pkg/clustermanager/eksa_installer_test.go

```diff
@@ -252,6 +253,7 @@ func TestSetManagerFlags(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
+			os.Unsetenv(features.FullLifecycleAPIEnvVar)
```

Comment:
I'm assuming this is to prevent the current env from where we run the test to affect the results of the test.

If that's the case, (nit) can we move this line to the beginning of the test function? That removes ambiguity about this line trying to cleanup between sub-tests vs setting up the environment appropriately before the whole test.

Reply:
I put it here because it should be checked and fixed on a per-test basis. Each test should ensure the environment is pristine, regardless of what other tests do. That property is lost if its moved to the beginning of the test function.